### DISCUSSION TITLE
BUG: Onboarding page stuck on the first page

### DISCUSSION THREAD

**Discussion by lf-** [community/community#88620] [GitHub Education]
### Hi everyone,

What I have done: browsed to https://education.github.com/onboardings/new, then clicked "Skip this question" or even filled out the form and clicked continue. This has been observed in both Chromium and Firefox, including with all ad blockers and suchlike disabled. This appears to be a bug.

I have also tried https://education.github.com/discount_requests, which is broken because the form POST request is missing the user type field (on account of not being propagated from the UI).

It appears there has been a critical lack of practical testing of either the current or previous version of the signup flow recently.

What I expected to happen: any change

What happened: no UI change.

Other observations:

Errors in dev tools, related to someone forgetting to have jquery, seemingly?

```
Uncaught SyntaxError: Identifier 'formShown' has already been declared
skip_to_main-d9cc808460d7eec555e080662196aaf66fa025756ed9e09070eda4ad2c3641e9.js:1 Uncaught ReferenceError: $ is not defined
    at skip_to_main-d9cc808460d7eec555e080662196aaf66fa025756ed9e09070eda4ad2c3641e9.js:1:1
onboarding-99b491364969e0e088e335cea210ff32cfe6a468cce630614a60dc97ae923da6.js:1 Uncaught ReferenceError: $ is not defined
    at onboarding-99b491364969e0e088e335cea210ff32cfe6a468cce630614a60dc97ae923da6.js:1:1
```

No requests in dev tools on this flow.

![image](https://github.com/community/community/assets/6652840/e56f07c3-d7f3-4448-957c-eaa141effeaf)

**Comment by messard1** [comment]
I'm having the same problem. I think it's down for everyone.

**Reply by Mostafa-Ayman1** [reply]
yea i have the same problem

**Reply by messard1** [reply]
I just refreshed it and it works now.

**Reply by mishmanners** [reply]
Glad to hear it's working now 😄
